Parsers/SysmonParser.txt

```diff
@@ -1,5 +1,5 @@
 // KQL Sysmon Event Parser
-// Last Updated Date: Sept 4, 2019
+// Last Updated Date: Nov 25, 2019
 // Sysmon Version : 10.2.0.0
 //
 // Sysmon Instructions:
```

Comment:
After thinking about this further, I think we will need multiple parser versions as changes like this occur, but where older versions of sysmon are still deployed and producing data with the older schema.  Let's look at creating a new file with a version in the file name.